src/generateSdk/sdkGenerator/PythonSdkGenerator.ts

```diff
@@ -161,6 +199,47 @@ class SdkGenerator implements SdkGeneratorInterface {
 
     return generateSdkOutput;
   }
+
+  getParamType(elem: Node): string {
+    if (elem.type === AstNodeType.CustomNodeLiteral) {
+      return (elem as CustomAstNodeType).rawValue;
+    } else if (elem.type === AstNodeType.StringLiteral) {
+      return "str";
+    } else if (elem.type === AstNodeType.IntegerLiteral) {
+      return "int";
+    } else if (elem.type === AstNodeType.FloatLiteral || elem.type === AstNodeType.DoubleLiteral) {
+      return "float";
+    } else if (elem.type === AstNodeType.BooleanLiteral) {
+      return "bool";
+    } else if (elem.type === AstNodeType.AnyLiteral) {
+      return "Any";
```

Comment:
Is there `Any` in python?

Reply:
yes, as python introduced type hints, Any is available, but it has to be imported from `typing`